platform/android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/http/HttpRequestUrl.java

```diff
@@ -10,7 +10,10 @@ private HttpRequestUrl() {
   }
 
   /**
-   * Adapts a resource request url based on the host and query size.
+   * Adapts a resource request url based on the host, query size, and offline requirement.
+   * The Maps SDK SKU token is not included with offline tile requests because offline requests
```

Comment:
Thanks @langsmith! The inline documentation for this change totally works, but I have a slight preference to keep things consistent across platforms. I also slightly prefer @friedbunny's wording below because it links to the pricing page and explains the impact of not sending SKU tokens with the request without mentioning "SKU tokens" explicitly, as we don't really expect customers to know what a SKU token is or what it does.

>  Mapbox resources downloaded for offline use are subject to separate Vector Tile and
 Raster Tile API pricing and are not included in the Maps SDK’s “unlimited” requests.
 See <a href="https://www.mapbox.com/pricing/">our pricing page</a> for more information.

Reply:
Added in https://github.com/mapbox/mapbox-gl-native/pull/15363/commits/99a613155a609f3a023b3fd4a94a78876f9d1b59